### PR TITLE
[mqtt.homeassistant] sensors with a state_class are numeric

### PR DESCRIPTION
**Pull request by ccutrer**
see reference in code comment, but a measurement sensor is assumed to be numeric, even if it doesn't have a unit

### CODE REVIEW

File: bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Sensor.java [hunk 1]
```diff
@@ -70,9 +72,14 @@ public Sensor(ComponentFactory.ComponentConfiguration componentConfiguration) {
 
         Value value;
         String uom = channelConfiguration.unitOfMeasurement;
+        String sc = channelConfiguration.stateClass;
 
         if (uom != null && !uom.isBlank()) {
             value = new NumberValue(null, null, null, UnitUtils.parseUnit(uom));
+        } else if (sc != null && !sc.isBlank()) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Sensor.java:79]: No need to check for "None"?

Comment (ccutrer) [reply bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Sensor.java:79]: No. `None` in the HASS docs isn't a literal string, but the Python `None`, which is equivalent to `null` in Java and JSON.

Comment (lolodomo) [reply bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Sensor.java:79]: Ok

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**